src/providers/data_providers/googlePlacesBusinessImageProvider.ts

```diff
@@ -8,18 +8,28 @@ export class GooglePlacesBusinessImageProvider extends BaseBusinessImageProvider
 		const googlePlacesApi = await new GooglePlacesApiUtils(this.config).init();
 		return googlePlacesApi
 			.getPlaceID(businessInfo.address, businessInfo.name)
-			.then((id) => googlePlacesApi.getPlaceDetails(id))
-			.then((photoRefs) =>
-				Promise.all(photoRefs.map((photoRef) => googlePlacesApi.getPhoto(photoRef)).filter((x) => x))
-			)
-			.then((photoUrls) =>
-				photoUrls.map(
-					(url: any) =>
-						<Image>{
-							source: url,
-							provider: ImageProviderName.google_places,
-						}
+			.then((id) => googlePlacesApi.getPlacePhotoRefs(id))
+			.then((photoRefs: Array<string>) =>
+				Promise.all(
+					photoRefs.map((photoRef) => {
+						return googlePlacesApi.getPhoto(photoRef);
+					})
 				)
-			);
+			)
+			.then((photoUrls: Array<string>) =>
```

Comment:
This type annotation isn't actually correct; It should be `Array<string | undefined>`, which shouldn't be necessary.